joint_limits:
  a1_joint:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 0.205
    has_acceleration_limits: true
    max_acceleration: 0.005
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
  a2_joint:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 0.204
    has_acceleration_limits: true
    max_acceleration: 0.005
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
  a3_joint:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 0.204
    has_acceleration_limits: true
    max_acceleration: 0.005
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
  a4_joint:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 0.27
    has_acceleration_limits: true
    max_acceleration: 0.0067
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
  a5_joint:
    has_position_limits: true
    min_position: -3.1415
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 0.26
    has_acceleration_limits: true
    max_acceleration: 0.0067
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
  a6_joint:
    has_position_limits: false
    has_velocity_limits: true
    max_velocity: 0.009
    has_acceleration_limits: true
    max_acceleration: 0.36
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
